_posts/16-07-01-Components.md

```diff
@@ -48,3 +55,9 @@ components best decoupled from the Laravel framework are listed above._
 [Eloquent ORM]: https://github.com/illuminate/database
 [Queue]: https://github.com/illuminate/queue
 [Illuminate components]: https://github.com/illuminate
+[Collection]: https://github.com/cakephp/collection
+[Database]: https://github.com/cakephp/database
+[Datasource]: https://github.com/cakephp/datasource
+[Event]: https://github.com/cakephp/event
+[I18n]: https://github.com/cakephp/i18n
+[ORM]: https://github.com/cakephp/orm
```

Comment:
i think 1 line would be enough ( `[CakePHP Components]: https://github.com/cakephp` ).